css/index.css

```diff
@@ -370,20 +370,87 @@
   border-color: #000;
 }
 
-#contact{
-  overflow: hidden;
+@media (min-width: 769px) {
+  #contact{
+    border-style: solid;
+    border-color: #fff;
+    border-radius: 100px / 70px;  /* chosen different vertical and horizontal radius for smooth border */
+    overflow: hidden;
+    display: flex;
+    justify-content: space-between;
+    gap: 20px;
+    padding: 60px 30px;
+    margin: 100px 20px;
+  }
+  
+  .contactLeftSection{
+    align-items: center;
+    justify-content: center;
+    flex: 40%;
+    padding: 20px;
+  }
+  
+  .contactRightSection{
+    align-items: top;
+    justify-content: right;
+    flex: 60%;
+    padding: 20px;
+  }
+  
+  #titleContact{
+    font-family: "Montserrat", sans-serif;
+    text-transform: uppercase;
+    font-weight: 200;
+    font-size: 60px;
+    text-align: center;
+  }
+  
+  .mailText{
+    font-family: "Lucida Console", "Courier New", monospace;
+    padding-bottom: 15px;
+    text-align: right;
+  }
+  
+  .email{
+    font-style: italic;
+  }
 }
-#contactMap{
-  display: block;
-  width: 60%;
-  float: left;
+
+@media (max-width: 769px) {
+  #contact{
+    overflow: hidden;
+    padding: 10% 5%;
```

Comment:
The relative margin and padding are making the contact section appear quite shrinked on mobile resolution. It is better to use pixel values for this.
![image](https://github.com/user-attachments/assets/a8eacd86-27bd-4dc6-9b67-de46b97c416c)

Reply:
is it just me, or is it the case that stuff is not properly centered?

Reply:
Did the changes, also fixed the issue of things not being centered.